Makefile

```diff
@@ -313,6 +313,17 @@ proto-api-v3-all:
 		protoc-gen-swagger/options/openapiv2.proto \
 		gogoproto/gogo.proto
 
+
+.PHONY: proto-storage-all
+proto-storage-all:
```

Comment:
api_v3 has the http wrapper and swagger definition as well - do we want the same here?